src/Service/Controllers/HealthController.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.DataApiBuilder.Service.HealthCheck;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Azure.DataApiBuilder.Service.Controllers
+{
+    [ApiController]
+    [Route("/health")]
+    public class HealthController : ControllerBase
+    {
+        public IHttpContextAccessor httpContextAccessor;
```

Comment:
```suggestion
        public IHttpContextAccessor HttpContextAccessor;
```

public properties need to be capitalized